Casks/font/font-n/font-nacelle.rb

```diff
@@ -0,0 +1,32 @@
+cask "font-nacelle" do
+  version "1.00"
+  sha256 "4e21fc1e22b15c4e1a7c520cf48ec5d47a5803beee21ac7edc0b205c340a4065"
+
+  url "https://dotcolon.net/files/fonts/nacelle_#{version.to_s.tr(".", "")}.zip"
```

Comment:
```suggestion
  url "https://dotcolon.net/files/fonts/nacelle_#{version.no_dots}.zip"
```

We have a related helper method.